rpcs3/Emu/RSX/RSXThread.h

```diff
@@ -720,7 +722,7 @@ namespace rsx
 		bool capture_current_frame = false;
 
 	public:
-		bool invalid_command_interrupt_raised = false;
+		u32 skip_methods_count = 0;
```

Comment:
This should have been a struct and avoided special codes like -1/umax. Improves readability.

Reply:
Used rsx->fifo_ctrl->skip_methods() directly as now it is a public member.